.github/src/readme.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+
+import { marked } from "marked";
+import yaml from "yaml";
+
+/**
+ * @param {string} markdown
+ * @param {import('github-script').AsyncFunctionArguments['core']} core
+ * @returns {Promise<Set<string>>} All input files for all tags
+ */
+export async function getInputFiles(markdown, core) {
+  const tokens = marked.lexer(markdown);
+
+  const yamlBlocks = tokens
+    .filter((token) => token.type === "code")
+    .map((token) => /** @type import("marked").Tokens.Code */ (token))
+    // Include default block and tagged blocks (```yaml $(tag) == 'package-2021-11-01')
+    .filter((token) => token.lang?.toLowerCase().startsWith("yaml"));
```

Comment:
The filtered yaml blocks would contain `swagger to sdk` one and `suppressions` which don't have 'input-file'.
[example](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/apimanagement/resource-manager/readme.md#swagger-to-sdk)

Reply:
Agreed, but what problem do you think this causes?  After parsing the YAML, we filter to only `obj["input-file"]`.